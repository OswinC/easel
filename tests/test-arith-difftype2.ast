int a;
float b;
pix c;
a = 6;
b = 3.;
c = 2;
print(a + c);
print(a / c);
printfl(a + b);
printfl(a / b);
printfl(b + c);
printfl(b / c);
printp(a + c);
printp(a / c);
