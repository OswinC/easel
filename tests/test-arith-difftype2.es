int a;
float b;
pix c;

a = 6;
b = 3.0;
c = 2;


/*print(a+b);
print(a/b);*/
/*print(a+c);*/
print(a/c);
/*print(b+c);*/
/*print(b/c);*/
printfl(a+b);
printfl(a/b);
/*printfl(a+c);*/
/*printfl(a/c);*/
printfl(b+c);
printfl(b/c);

/*printp(a+b);
printp(a/b);*/
printp(a+c);
printp(a/c);
/*printp(b+c);
printp(b/c);*/
