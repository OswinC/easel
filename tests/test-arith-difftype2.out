3
9.000000
2.000000
5.000000
1.500000
#8
#3
